bigquery_etl/cli/query.py

```diff
@@ -1,6 +1,7 @@
 """bigquery-etl CLI query command."""
```

Comment:
Should we consider using a different namespace than `query` for this? It feels like it may cause some confusion since we're not checking the query, but were checking the data. Maybe a new namespace called `check` or `data_check` would help avoid this?

Reply:
I followed what was in the proposal which indicated  `query` (which also includes the `run` command).  I agree `check` isn't a great name, see point 2 in the above Summary.  I can update it to data_check.

Reply:
Sounds good,  I'll move it to a separate command.